From 404cac570ffc9367a3182467697316ee48dc3302 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 2 Nov 2023 22:26:02 +0100
Subject: [PATCH 0057/1695] plat-stm32mp1: syscfg: use U() macro

Use U() macro where applicable in stm32mp1_syscfg.c driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d6d1549231a1339c52098e4456085d1666244231 plat-stm32mp1: syscfg: use U() macro]
Change-Id: Iffe19c18d7034c6f7252ee2535c44165faa18f83
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from e287ddde9c70f316f49b868b7eccb29934de9d0d]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 .../arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c
index 281ae7caa..34bd68c10 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_syscfg.c
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: BSD-2-Clause
 /*
- * Copyright (c) 2019-2022, STMicroelectronics
+ * Copyright (c) 2019-2023, STMicroelectronics
  */
 
 #include <drivers/clk.h>
@@ -16,8 +16,8 @@
 /*
  * SYSCFG register offsets (base relative)
  */
-#define SYSCFG_CMPCR				0x20U
-#define SYSCFG_CMPENSETR			0x24U
+#define SYSCFG_CMPCR				U(0x20)
+#define SYSCFG_CMPENSETR			U(0x24)
 
 /*
  * SYSCFG_CMPCR Register
@@ -25,11 +25,11 @@
 #define SYSCFG_CMPCR_SW_CTRL			BIT(1)
 #define SYSCFG_CMPCR_READY			BIT(8)
 #define SYSCFG_CMPCR_RANSRC			GENMASK_32(19, 16)
-#define SYSCFG_CMPCR_RANSRC_SHIFT		16
+#define SYSCFG_CMPCR_RANSRC_SHIFT		U(16)
 #define SYSCFG_CMPCR_RAPSRC			GENMASK_32(23, 20)
-#define SYSCFG_CMPCR_ANSRC_SHIFT		24
+#define SYSCFG_CMPCR_ANSRC_SHIFT		U(24)
 
-#define SYSCFG_CMPCR_READY_TIMEOUT_US		1000U
+#define SYSCFG_CMPCR_READY_TIMEOUT_US		U(1000)
 
 /*
  * SYSCFG_CMPENSETR Register
-- 
2.47.2

